### PR TITLE
Add notes on impact of model inputs and reference training/fine-tuning/scorer docs from usage docs

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

File: doc/USING.rst [hunk 1]
```diff
@@ -54,6 +54,15 @@ There are several pre-trained model files available in official releases. Files
 
 Finally, the pre-trained model files also include files ending in ``.scorer``. These are external scorers (language models) that are used at inference time in conjunction with an acoustic model (``.pbmm`` or ``.tflite`` file) to produce transcriptions. We also provide further documentation on :ref:`the decoding process <decoder-docs>` and :ref:`how language models are generated <scorer-scripts>`.
 
+Important considerations on model inputs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The release notes include detailed information on how the released models were trained/constructed. Important considerations for users include the characteristics of the used training data and whether they match your intended use case. For acoustic models, an important characteristic is the demographic distribution of speakers. For language models, the sources of text used in their construction. If the data used for training the models does not align with your intended use case, it may be necessary to adapt or train new models in order to get good accuracy in your transcription results.
```
Comment (kdavis-mozilla) [inline doc/USING.rst:60]: > For language models, the sources of text used in their construction.

I don't think this is conveying the idea you're looking to convey. For example, the sources of two texts could be Wikipedia, but the texts could be very different, e.g. one from simplified Wikipedia page talking about frogs and one from Wikipedia quotes quoting a PhD thesis on the toxins of Rhinella marina.

Maybe something like

> For language models, the texts should be similar to those of the expected use case.

is more to the point.

Comment (kdavis-mozilla) [inline doc/USING.rst:60]: Word order here...

> Important considerations for users include the characteristics of the **used training data** and...

is a bit off it should be

> Important considerations for users include the characteristics of the **training data used** and...

Comment (kdavis-mozilla) [inline doc/USING.rst:60]: Also shouldn't we be saying "scorer" instead of "language model" here?

**Comment by reuben** [comment]
Thanks!

**Pull request merged**